### PR TITLE
EA Telemetry: Add meta field to ea-bootstrap responses.

### PR DESCRIPTION
**Pull request by cawthorne**
Adds the required meta field to the EAs which use `ea-bootsrap`, to enable the EA Telemetry.

### CODE REVIEW

**Comment by mxiao-cll** [comment]
👍

**Review by alecgard — DISMISSED** [review]

File: packages/sources/ipfs/test/integration/write.test.ts [hunk 1]
```diff
@@ -29,6 +29,17 @@ afterAll(() => {
   nock.enableNetConnect()
 })
 
+describe('dummy test', () => {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  it('dumm test case', async () => {})
+})
+
+/*
+NOTE: Commenting out as the test is failing and the EA isn't being used.
```
Comment (mxiao-cll) [inline packages/sources/ipfs/test/integration/write.test.ts:38]: Should just remove the EA

**Review by mxiao-cll — APPROVED** [review]

File: packages/sources/ipfs/test/integration/write.test.ts (hunk 1 above)
Comment (cawthorne) [reply packages/sources/ipfs/test/integration/write.test.ts:38]: Did consider this. There are quite a lot of EAs that could be removed probably. I wondered there might be a reason they'd been left in.

Comment (cawthorne) [reply packages/sources/ipfs/test/integration/write.test.ts:38]: Could remove in a cleanup PR later if needed.

**Pull request merged**